[git-webkit] base-commit for PRs should control remote

JonWBedard opened the pull request:
#### c85ae4314641181e32bd6c83d0a41dc36991d885
<pre>
[git-webkit] base-commit for PRs should control remote
<a href="https://bugs.webkit.org/show_bug.cgi?id=242662">https://bugs.webkit.org/show_bug.cgi?id=242662</a>
&lt;rdar://96914116&gt;

Reviewed by David Kilzer.

* Tools/Scripts/libraries/webkitscmpy/setup.py: Bump version.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py: Ditto.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py:
(Git.source_remotes): Return a list of source remotes, which are distinct from individual&apos;s
forked remotes.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py:
(PullRequest.pull_request_branch_point): If no remote is defined, use the branch_point and
project source_remotes to determine which remote to use.
(PullRequest.add_comment_to_reverted_commit_bug_tracker): Use default_remote instead of &apos;origin&apos;.
(PullRequest.create_pull_request): Ditto.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/git_unittest.py:

Canonical link: <a href="https://commits.webkit.org/252475@main">https://commits.webkit.org/252475@main</a>
</pre>

ddkilzer reviewed (approved):
And I left the "rs=me" comment in the wrong place.  See above.

JonWBedard committed:
[git-webkit] base-commit for PRs should control remote
https://bugs.webkit.org/show_bug.cgi?id=242662
<rdar://96914116>

Reviewed by David Kilzer.

* Tools/Scripts/libraries/webkitscmpy/setup.py: Bump version.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py: Ditto.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py:
(Git.source_remotes): Return a list of source remotes, which are distinct from individual's
forked remotes.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py:
(PullRequest.pull_request_branch_point): If no remote is defined, use the branch_point and
project source_remotes to determine which remote to use.
(PullRequest.add_comment_to_reverted_commit_bug_tracker): Use default_remote instead of 'origin'.
(PullRequest.create_pull_request): Ditto.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/git_unittest.py:

Canonical link: https://commits.webkit.org/252475@main
